### ISSUE TITLE
Pytorch asks to flatten the paramters when I'm using higher for meta-learning a simple seq2seq LSTM model. 

### ISSUE THREAD

**Issue by n-askarian** [facebookresearch/higher#65]
Hi dear authors,
Firstly, thanks for this great and easy-to-use tool. 
I am trying to use the higher package for meta-learning a simple seq2seq model consists of two PyTorch vanilla LSTMs as the encoder and the decoder. During training, after two iterations I receive the following warning for an infinite number of times until I make to stop the experiment.   
_"_/pytorch/aten/src/ATen/native/cudnn/RNN.cpp:1269: UserWarning: RNN module weights are not part of a single contiguous chunk of memory. This means they need to be compacted at every call, possibly greatly increasing memory usage. To compact weights again call flatten_parameters."_ 
As suggested in the warning I used **flatten_paramters** function at the start of the **forward** function of the encoder and the decoder. But nothing changed. 
It is also worth to mention that the model is trained with no issue in a standard supervised learning setting. 
Can you figure out what would be the reason?

Thanks in advance.

**Comment by egrefen** [comment]
Hello. Can you please post a complete (and preferably minimal) code snippet that replicates this warning so we can investigate?

**Comment by n-askarian** [comment]
Thanks for the reply.
Here is a self-contained code snippet that reproduces the warning when running on GPU. The warning is produced for every forward pass to LSTM. I ran the code using Python 3.6 and both Pytorch 1.3 and 1.5 and received the same warning.


```
import torch
import higher
import torch.cuda
import torch.nn as nn
import torch.nn.functional as F

device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")

class Seq2Seq(nn.Module):
    def __init__(self,
        encoder_vocab_size=93,
        decoder_vocab_size=44,
        wordvec_dim=300,
        hidden_dim=256,
        rnn_num_layers=2,
        rnn_dropout=0,
        ):
        super(Seq2Seq, self).__init__()
        self.encoder_rnn = nn.LSTM(wordvec_dim, hidden_dim, rnn_num_layers, dropout=rnn_dropout, batch_first=True)
   
    def forward(self, x):
        embed = torch.zeros(15, 43, 300).to(device)
        out, _ = self.encoder_rnn(embed)
        return out

def main():
    net = Seq2Seq().to(device)
    xs = torch.randint(0, 100, (3, 43)).to(device)
    fmodel = higher.monkeypatch(net)
    output = fmodel(xs)
     
if __name__ == '__main__':
    main()
```

**Comment by egrefen** [comment]
Thanks for the snippet. I not this doesn't yield warnings on CPU, just GPU, but I can reproduce this. I'll look into it.

**Comment by egrefen** [comment]
This should be addressed by #67. Re-open if it doesn't address your problem.
